tests/app/dao/test_service_permissions_dao.py

```diff
@@ -40,22 +41,31 @@ def test_fetch_service_permissions_gets_service_permissions(
         sp.service_id == service_without_permissions.id for sp in service_permissions
     )
     assert all(
-        sp.permission in [INTERNATIONAL_SMS_TYPE, SMS_TYPE]
+        sp.permission
+        in {
+            ServicePermissionType.INTERNATIONAL_SMS,
+            ServicePermissionType.SMS,
+        }
```

Comment:
I'm assuming this was one of those other odd test failures that required a data type change?

Reply:
Yes